README.rst

```diff
@@ -62,8 +62,8 @@ Initial setup and installation:
   solr config, using the core/collection names for development and
   testing that you configured in local settings::
 
-    solr create -c SOLR_CORE -d solr_conf
-    solr create -c SOLR_TEST_CORE -d solr_conf
+    solr create -c ppa -d solr_conf
+    solr create -c ppa-test -d solr_conf
```

Comment:
Reasonable! Then this will match exactly what's in the sample local settings, right? That's good.

Reply:
Correct, it's one fewer thing to change